src/cmd/services/m3coordinator/downsample/downsampler_test.go

```diff
@@ -65,22 +57,25 @@ const (
 )
 
 func TestDownsamplerAggregationWithAutoMappingRules(t *testing.T) {
-	testDownsampler := newTestDownsampler(t, testDownsamplerOptions{
-		autoMappingRules: []MappingRule{
+	testDownsampler, err := NewTestDownsampler(TestDownsamplerOptions{
```

Comment:
Won't this have to regenerate pools for each run? May be better to keep built downsamplers to speed up the tests